src/app/content/components/BookBanner.tsx

```diff
@@ -1,17 +1,17 @@
 import { HTMLAnchorElement, HTMLDivElement } from '@openstax/types/lib.dom';
-import React, { Component } from 'react';
```

Comment:
why did you refactor this whole component? its introducing a lot of potential for issues and i'm not sure there was a reason to do it

Reply:
I thought that it would be easier to make make `BookBanner` a functional component and use `hook` as it's not as easy to pass the services a `prop`.